sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -36,6 +36,13 @@ public class AzureMonitorOptions : ClientOptions
         /// </summary>
         public bool DisableOfflineStorage { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ratio of telemetry items to be sampled. The value must be between 0.0F and 1.0F, inclusive.
+        /// For example, specifying 0.4 means that 40% of traces are sampled and 60% are dropped.
+        /// The default value is 1.0F, indicating that by default, all telemetry items are sampled.
```

Comment:
Don't need the word "default" twice in the same sentence.

Could write this as "The default value is 1.0F, indicating that 100% of telemetry items are sampled."